googleapiclient/http.py

```diff
@@ -84,26 +88,90 @@ def _retry_request(http, num_retries, req_type, sleep, rand, uri, method, *args,
     resp, content - Response from the http request (may be HTTP 5xx).
   """
   resp = None
+  content = None
+  exception = None
   for retry_num in range(num_retries + 1):
     if retry_num > 0:
-      sleep(rand() * 2**retry_num)
-      LOGGER.warning(
-          'Retry #%d for %s: %s %s%s' % (retry_num, req_type, method, uri,
-          ', following status: %d' % resp.status if resp else ''))
+      # Sleep before retrying.
+      sleep_time = rand() * 2 ** retry_num
+      LOGGER.warning('Sleeping %.2f seconds before retry %d of %d for %s: '
```

Comment:
If you line-wrap after the opening parenthesis, can you avoid the implicit string concatenation? Always avoid implicit string concatenation whenever you can.